midend/expandLookahead.cpp

```diff
@@ -41,16 +41,16 @@ void DoExpandLookahead::expandSetValid(const IR::Expression* base, const IR::Typ
 const IR::Expression* DoExpandLookahead::expand(
     const IR::PathExpression* base, const IR::Type* type, unsigned* offset) {
     if (type->is<IR::Type_Struct>() || type->is<IR::Type_Header>()) {
-        auto vec = new IR::Vector<IR::Expression>();
+        auto vec = new IR::ListExpression({});
```

Comment:
There are a *lot* of instances of `new IR::ListExpression({})`. You should probably mark `ListExpression::components` as optional.